scripts/plugin/package_plugin.py

```diff
@@ -0,0 +1,106 @@
+
+import shutil
+import os
+from os import path
+
+from distutils import dir_util, file_util
+
+plugin_toml_template = '''
+schema_version = 1
+name = "dcos-core-cli"
+
+[[commands]]
+name = "job"
+path = "bin/dcos{0}"
```

Comment:
For now, until we decide the go rewrite is ready, I would keep the python binary.

Reply:
👍
This brings up the bigger point of whether or not I should be packaging the Go part if nothing is using it. I think I'll take it out for now.